Gordon360/ApiControllers/HousingController.cs

```diff
@@ -25,10 +25,71 @@ public HousingController()
             _accountService = new AccountService(_unitOfWork);
         }
 
+        /// <summary>Check if the currently logged in user is authorized to view the housing staff page for applications</summary>
+        /// <returns></returns>
+        [HttpGet]
+        [Route("staff")]
```

Comment:
I assume this is callable even if it doesn't do much. I guess we could merge it as is and implement it further later so that we can use the frontend that calls this endpoint.

Reply:
It's an intententional placeholder